docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.paragraph.field_summary_box_content.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
```

Comment:
So this is an approach question, but why were we not able to use the NCIDS content block's field?